modules/library/jdbc/src/test/java/org/geotools/jdbc/JDBCDateTest.java

```diff
@@ -40,11 +40,11 @@ public void testFiltersByDate() throws Exception {
         df.setTimeZone( TimeZone.getTimeZone("PST"));
         
         //less than
-        Filter f = ff.lessOrEqual( ff.property( aname("d") ), ff.literal( df.parse("2009-28-06")
-         ) );
+        Filter f = ff.lessOrEqual( ff.property( aname("d") ), ff.literal("2009-06-28") );
```

Comment:
If you use a date literal in both parts of the test they are identical - so I assumed one of them was testing with a string.